cloud-controller-manager/do/loadbalancers.go

```diff
@@ -432,6 +432,10 @@ func buildForwardingRules(service *v1.Service) ([]godo.ForwardingRule, error) {
 	certificateID := getCertificateID(service)
 	tlsPassThrough := getTLSPassThrough(service)
 
+	if (certificateID != "" || tlsPassThrough) && len(tlsPorts) == 0 {
+		tlsPorts = append(tlsPorts, 443)
```

Comment:
I feel like should only do this if port 443 is listed among the entryPorts.  Otherwise, this is moot.  It seems likely if the user forgot to include to include 443 in their `do-loadbalancer-tls-ports`, they also forgot it in their entryPorts, especially if they're using a certificateID.  

Alternatively, if they do provide entry port 80, `do-loadbalancer-certificate-id` and `do-loadbalancer-redirect-http-to-https` we could safely assume the port 80 should entry point target port should be really be used for port 443 and create it implicitly.